### ISSUE TITLE
terminology declarations in the DMRS RDF output? 

### ISSUE THREAD

**Issue by arademaker** [own-pt/delphin-rdf#35]
```
dmrs:h  rdf:type  dmrs:ScopalRelation .
dmrs:neq  rdf:type  dmrs:ScopalRelation .
dmrs:arg1  rdf:type  dmrs:Role .
dmrs:rstr  rdf:type  dmrs:Role .
```

Those triples should be in the vocabularies, right? Why we are producing them in the graph of the DMRS?

**Comment by yfaria** [comment]
Yes, they should and, in fact, they are (see https://github.com/own-pt/delphin-rdf/blob/master/vocabularies/dmrs.ttl#L77 for example of the `arg1`). 
The code generates them because  I wasn't able to find a list of possible roles and scopal relations in DMRS when the code was written. The roles are simpler because those come from MRS roles, so I knew the possibilities.

**Comment by arademaker** [comment]
Not sure if I understood, but what action we can take? Do you still really need to generate these triples? We do have arguments for generating those triples in the DMRS generation of a given semantic representation?
